capstone/src/main/java/com/google/sps/servlets/UsersServlet.java

```diff
@@ -40,96 +40,70 @@
     "SELECT user_info.id, user_info.email,  user_info.full_name, " +
     "user_info.user_name, university.name " +
     "FROM user_info JOIN university ON university.id = user_info.university_id " +
     "WHERE user_info.email = ?";
 
-  private final String CREATE_NEW_USER_QUERY =
-    "INSERT INTO user_info (user_name, full_name, email, verified, university_id) VALUES (?, ?, ?, true, ?)";
-
-  private final String URL_TO_REDIRECT_AFTER_USER_LOGS_OUT = "";
-  private final String URL_TO_REDIRECT_AFTER_USER_LOGS_IN = "";
-
-  /**
-   * This method gets the related information associated with a user based on the email provided by the userService parameter.
-   * @param pool It is a DataSource object that serves to interact with our database connection.
-   * @param userService This is a UserService object that holds information about the current user.
-   * @param get_user_row_query This is an SQL statement to query our database for the needed user's information.
-   * @param response A HttpServletResponse object that holds the information that will be sent back to the user.
-   * @return HashMap<String, String> saves a key-value pair of all neccessary user details derived.
-   */
-  public HashMap<String, String> getUserDetails(
-    DataSource pool,
-    UserService userService,
-    String get_user_row_query,
-    HttpServletResponse response
-  )
-    throws SQLException, IOException {
-    HashMap<String, String> userDetails = new HashMap<>();
-
-    Boolean isUserLoggedIn = userService.isUserLoggedIn();
-    if (!isUserLoggedIn) {
-      String user_status = Boolean.toString(isUserLoggedIn);
-      userDetails.put("user_status", user_status);
-      userDetails.put("loginUrl",  userService.createLoginURL(URL_TO_REDIRECT_AFTER_USER_LOGS_IN));
-      userDetails.put("status_code", "401");
-      return userDetails;
-    }
-    return checkForStudentEmail(
-      pool,
-      userService,
-      get_user_row_query,
-      response
-    );
-  }
-
   /**
-   * This method checks if the email supplied is a student email by confirming the '.edu' tld.
+   * This method checks if the current user is already registered in our database.
    * @param pool It is a DataSource object that serves to interact with our database connection.
    * @param userService This is a UserService object that holds information about the current user.
    * @param get_user_row_query This is an SQL statement to query our database for the needed user's information.
    * @param response A HttpServletResponse object that holds the information that will be sent back to the user.
-   * @return HashMap<String, String> saves a key-value pair of all neccessary user details derived.
+   * @param request A HttpServletRequest object that holds the information sent by the user.
+   * @return Boolean
    */
-  public HashMap<String, String> checkForStudentEmail(
+  public Boolean IsUserRegistered(
     DataSource pool,
     UserService userService,
     String get_user_row_query,
```

Comment:
To clarify I do not think line 70 is need since every method in this class should be able to acess GET_USER_ROW_QUERY without it being declared in the method